README.md

```
@@ -8,37 +8,64 @@ It is a party information system for the bigscreen at [demoparties](https://en.w
 ## Quickstart
 
 We use `mise` to automatically manage all tool versions (Go, Node, etc.) and project tasks.
 
-```bash
-# 1. Install mise (if not already installed)
-curl https://mise.run | sh
+Please refer to `mise` documentation on how to install it.
 
-# 2. Setup the project (installs dependencies and starts infra)
-mise run setup
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)